hack/run-ci-e2e-test.sh

```diff
@@ -11,6 +11,9 @@ BYOH_NODE_COUNT_OPTION=""
 SKIP_NODE_DELETION=""
 WMCO_PATH_OPTION=""
 
+# WINDOWS_INSTANCES_DATA holds the windows-instances configMap data section
+declare -r WINDOWS_INSTANCES_DATA=${WINDOWS_INSTANCES_DATA:-}
```

Comment:
Why do you need to use `declare` given we have not be doing this before? I under stand the weak typesetting this allows but I don't see the advantage and would like us to be consistent.

Reply:
Using `declare` to ensure `$WINDOWS_INSTANCES_DATA` is read-only (` -r `) as a constant. [Before](https://github.com/openshift/windows-machine-config-operator/blob/0289df3c644482d7727a0f52f2082d4a8876cbe2/hack/run-ci-e2e-test.sh#L12) the usage aligns more as variables.

Reply:
There are many other variable where this applies to. I don't see this as being reason enough to introduce this.